core/src/sumeragi/main_loop.rs

```diff
@@ -258,6 +250,11 @@ impl Sumeragi {
             "Genesis transaction set contains invalid transactions"
         );
 
+        assert!(
+            block.rejected_transactions.is_empty(),
```

Comment:
This code is duplicate from the above.

Reply:
Good catch. Git be gitting. I'll remove it.